File: C#Integration/PokerProtocol/Commands/DisconnectCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using EricUtility;

namespace PokerProtocol.Commands
{
    public class DisconnectCommand : AbstractCommand
    {
        protected override string CommandName
        {
            get { return DisconnectCommand.COMMAND_NAME; }
        }
        public static string COMMAND_NAME = "DISCONNECT";


        public DisconnectCommand(StringTokenizer argsToken)
        {
        }

        public DisconnectCommand()
        {
        }
    }
}

File: trunk/JAVA/PokerProtocol/src/bluffinmuffin/protocol/commands/DisconnectCommand.java
package bluffinmuffin.protocol.commands;

import java.util.StringTokenizer;


public class DisconnectCommand extends AbstractCommand
{
    @Override
    protected String getCommandName()
    {
        return DisconnectCommand.COMMAND_NAME;
    }
    
    public static String COMMAND_NAME = "DISCONNECT";
    
    public DisconnectCommand(StringTokenizer argsToken)
    {
    }
    
    public DisconnectCommand()
    {
    }
}
